caluma/caluma_workflow/jexl.py

```diff
@@ -17,11 +37,21 @@ def visit_Transform(self, transform):
 
 class TaskValidatingAnalyzer(ValidatingAnalyzer):
     def visit_Transform(self, transform):
-        if transform.name == "tasks" and not isinstance(transform.subject.value, list):
-            yield f"{transform.subject.value} is not a valid list of tasks."
-
-        if transform.name == "task" and not isinstance(transform.subject.value, str):
-            yield f"{transform.subject.value} is not a valid task slug."
+        value = parse_literal(transform.subject.value)
+
+        if transform.name == "tasks":
```

Comment:
I think the validation should be moved to the transform methods themselves, not in the generic visit method

Reply:
Could you give an example on how you'd imagine that? The transform method is for evaluation and should not include validation since we can't evaluate at the time we validate..

Reply:
Yeh you're right - confused analyzer vs evaluation here.